js/index.js

```diff
@@ -38,5 +38,43 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
+const logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+// Nav Bar
+const navBar = document.querySelectorAll("a");
+navBar[0].textContent = siteContent["nav"]["nav-item-0"];
```

Comment:
If you wanna push yourself, assign the values using `forEach` or use old school `for` loop